src/Qowaiv/Diagnostics/Contracts/CollectionMutationAttribute.cs

```diff
@@ -0,0 +1,8 @@
+namespace Qowaiv.Diagnostics.Contracts;
+
+/// <summary>To mark a method explicitly as impure. Methods decorated with
```

Comment:
Comment is for impure instead of collectionmutation.

Reply:
That is not true. It states what to expect of the return value.